packages/merge-styles/src/extractStyleParts.ts

```diff
@@ -1,28 +1,29 @@
 import { IStyle, IStyleBaseArray } from './IStyle';
 import { Stylesheet } from './Stylesheet';
-import { ShadowConfig, isShadowConfig } from './shadowConfig';
+import { isShadowConfig } from './shadowConfig';
 
 /**
  * Separates the classes and style objects. Any classes that are pre-registered
  * args are auto expanded into objects.
  */
 export function extractStyleParts(
-  shadowConfig?: ShadowConfig,
+  sheet: Stylesheet,
   ...args: (IStyle | IStyle[] | false | null | undefined)[]
 ): {
   classes: string[];
   objects: IStyleBaseArray;
 } {
   const classes: string[] = [];
   const objects: {}[] = [];
-  const stylesheet = Stylesheet.getInstance(shadowConfig);
+  const stylesheet = sheet;
 
   function _processArgs(argsList: (IStyle | IStyle[])[]): void {
     // for (const arg of argsList) {
-    const startIndex = isShadowConfig(argsList[0]) ? 1 : 0;
+    // const startIndex = isShadowConfig(argsList[0]) ? 1 : 0;
```

Comment:
_nit:_ remove this line

```suggestion
    // const startIndex = isShadowConfig(argsList[0]) ? 1 : 0;
```